Automattic/wp-calypso
# Assembler - Device Switcher - Fixed device viewport

Related to #77547

## Proposed Changes

* Added `isFixedViewport` prop to use a fixed viewport per device
  * computer `1080px`
  * tablet `782px`
  * phone `480px`
* Added `isShowFrameShadow` prop to hide the shadow
* Scaled the action bar and border on hovered patterns


https://github.com/Automattic/wp-calypso/assets/1881481/2236405f-c989-4269-b496-8cd5ec3d9eca



### Follow-up issues
- Fix the height of patterns with 100vh so when a category is toggled or the window resized they use 100% viewport height.  

## Testing Instructions



- Click on the Calypso Live link in the first comment
- Create a new site with any plan
- Continue until the Design Picker
- Scroll down and click `Start designing`
- Click `Homepage` and add patterns
- Verify that the large preview is scaled for each device type
- Verify that is scaled after resizing the window or toggling a category to hide the patterns panel
- Test patterns from the "Link in bio" category to test that they use the 100% viewport height 


## Pre-merge Checklist
